main.go

```diff
@@ -36,15 +48,26 @@ func main() {
 	log.Infof("Running analyze")
 
 	analyzeCmd := command.New("flutter", append([]string{"analyze"}, additionalParams...)...).
-		SetStdout(os.Stdout).
-		SetStderr(os.Stderr).
 		SetDir(cfg.ProjectLocation)
 
 	fmt.Println()
 	log.Donef("$ %s", analyzeCmd.PrintableCommandArgs())
 	fmt.Println()
 
-	if err := analyzeCmd.Run(); err != nil {
+	out, err := analyzeCmd.RunAndReturnTrimmedCombinedOutput()
+	if err != nil {
+		if errorutil.IsExitStatusError(err) {
```

Comment:
NOTE This could be  `if errorutil.IsExitStatusError(err) && !hasAnalyzeError(out)` or add failf after if !hasAnalyzeError block